src/fontinfo.rs

```diff
@@ -610,6 +610,11 @@ impl FontInfo {
         }
     }
 
+    /// Returns `false` if this FontInfo has any non-default value, and `true` otherwise.
+    pub fn is_empty(&self) -> bool {
```

Comment:
What if the fontinfo *intentionally* has values that just happen to all be default?

Reply:
Then the fontinfo.plist is not written out AFAIK :)

Reply:
> that have been chosen by norad, and have nothing to do with the spec

Hm which ones? The values should assume the default values (None or something else) as laid out in https://unifiedfontobject.org/versions/ufo3/fontinfo.plist/.